_includes/head.html

```diff
@@ -2,6 +2,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="shortcut icon" type="image/png" href="http://www.pelagicore.com/favicon.png?v=2"/>
```

Comment:
This will make problems once we move on to https because it loads a http resource. I would be better if you downloaded that favicon and put it where we have the other images.